creator-node/src/routes/users.js

```diff
@@ -180,8 +187,24 @@ module.exports = function (app) {
     } catch (e) {
       // Swallow error, leave syncInProgress unset
     }
+    response.syncInProgress = syncInProgress
+
+    // Return CIDSkipInfo if requested
+    if (returnSkipInfo && cnodeUser) {
+      const CIDs = await models.File.findAll({
```

Comment:
i had same concern here - i read about this a bit, and seems like a count would not improve much since it will still have to iterate over same number of rows (confirmed this myself - the select actually was 3-4x faster than count, maybe since it isn't doing the additional compute)

Reply:
Surprised that was the result of your test!
https://stackoverflow.com/questions/15912749/is-count-faster-than-pulling-the-records-and-counting-in-code

You also have the side effect of passing a lot of data back from the DB (which could be outside of the container as well and would incur network cost). Since generally there could be a lot of CIDs for a given user, it seems best to not send that data back & forth (though not totally sure how often we will turn this flag on)?

This should be doable with a query like, which I think would be better?

```
SELECT 
  COUNT(CASE WHEN skipped = true then 1 ELSE NULL END) as "skipped",
  COUNT(*) as "total"
FROM "files"
WHERE...
```

Reply:
good point i didn't account for doing the count in code vs query or time to send data over. you right, given that it would probably make more sense to do counts in query, although the improvement would be minor

also yeah this will maybe be a future input into reconfig so nodes will request this from other nodes - never user facing

still, i'll make this change bc no reason not to